modules/deeparg/downloaddata/main.nf

```diff
@@ -11,10 +11,11 @@ process DEEPARG_DOWNLOADDATA {
     We have to force singularity to run with --fakeroot to allow reading of a problematic file with borked read-write permissions in an upstream dependency (theanos).
     This flag may not be available on all systems and may be considered a security problem. so please document and /or warn for this in your pipeline!
     */
-    containerOptions { "${workflow.containerEngine}" == 'singularity' ? '--fakeroot' : '' }
+    containerOptions { "${workflow.containerEngine}" == 'singularity' ? "-B ${dummy}:/usr/local/lib/python2.7/site-packages/Theano-0.8.2-py2.7.egg-info/PKG-INFO" : '' }
```

Comment:
This is a workaround for Singularity. Nf-core starts Docker with user emulation (not root), does that also cause problems?

Reply:
It didn't seem too :shrug: